build-parent/pom.xml

```diff
@@ -971,6 +971,24 @@
                             </externalDocumentationLinks>
                         </configuration>
                     </plugin>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-jar-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <id>dokka-jar</id>
+                                <phase>package</phase>
+                                <goals>
+                                    <goal>jar</goal>
+                                </goals>
+                                <configuration>
+                                    <classifier>javadoc</classifier>
+                                    <classesDirectory>${project.build.directory}/dokka</classesDirectory>
+                                    <skipIfEmpty>true</skipIfEmpty>
+                                </configuration>
+                            </execution>
+                        </executions>
+                    </plugin>
```

Comment:
@famod, @gsmet do you know anything about the necessity of this?

Reply:
AFAICS the `dokka` goal (unlike the `javadocJar` goal) is not creating a javadoc jar, which is compensated by this jar-plugin execution.

Looks like a limitation of the plugin: https://kotlin.github.io/dokka/1.7.10/user_guide/maven/usage/ (see goal descriptions and config options)

Reply:
Gotcha, thanks